### PR TITLE
[tensorflow][build] Port over TF1.13 Dockerfiles

### PR DESCRIPTION
**Pull request by arjkesh**
*Issue #, if available:*

## PR Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]

*Description:*
Releasing TF 1.13.1 with newer SM training toolkit

*Tests run:*
Running PR tests

*DLC image/dockerfile:*
tensorflow/training/docker/1.13.1/Dockerfile.cpu
tensorflow/training/docker/1.13.1/cu100/Dockerfile.gpu

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tensorflow/training/docker/1.13.1/py3/Dockerfile.cpu [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+           Pillow \
+           h5py==2.9.0 \
+           keras_applications==1.0.7 \
+           keras_preprocessing==1.0.9 \
+           keras==2.2.4 \
+           requests==2.24.0 \
+           "awscli<2" \
+           mpi4py==3.0.3 \
+           "sagemaker-tensorflow>=1.13,<1.14" \
+           "sagemaker-tensorflow-training>=10,<20" && \
```
Comment (arjkesh) [inline tensorflow/training/docker/1.13.1/py3/Dockerfile.cpu:96]: original dockerfile used sagemaker-training-container package - i.e. https://github.com/aws/sagemaker-tensorflow-training-toolkit/blob/master/docker/1.13.1/Dockerfile.gpu#L115

File: tensorflow/training/docker/1.13.1/py3/cu100/Dockerfile.gpu [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    Pillow \
+    h5py==2.9.0 \
+    keras_applications==1.0.7 \
+    keras_preprocessing==1.0.9 \
+    requests==2.24.0 \
+    keras==2.2.4 \
+    "awscli<2" \
+    mpi4py==3.0.3 \
+    "sagemaker-tensorflow>=1.13,<1.14" \
+    "sagemaker-tensorflow-training>=10,<20" \
```
Comment (arjkesh) [inline tensorflow/training/docker/1.13.1/py3/cu100/Dockerfile.gpu:131]: original dockerfile used sagemaker-training-container package https://github.com/aws/sagemaker-tensorflow-training-toolkit/blob/master/docker/1.13.1/Dockerfile.gpu#L115

**Review by ajaykarpur — APPROVED** [review]
`"sagemaker-tensorflow-training>=10,<20"` is correct.

**Comment by saimidu** [comment]
@arjkesh What needs to be done here to get this PR to a Review-Ready state?

**Pull request closed without merging**